mithril-test-lab/mithril-end-to-end/src/end_to_end_spec.rs

```diff
@@ -141,6 +142,25 @@ impl<'a> Spec<'a> {
                 .await?;
         }
 
+        // Verify that Cardano stake distribution artifacts are produced and signed correctly
+        {
```

Comment:
This should be executed only if the Cardano stake distribution is being signed.
You can see the implementation of `self.infrastructure.is_signing_cardano_transactions()` which is similar.

Reply:
Done ✅